### ISSUE TITLE
Waht's the meaning of this line?

### ISSUE THREAD

**Issue by yunyundong** [DagnyT/hardnet#25]
[labels = torch.cat([labels,datasets[i][1]])+torch.max(labels)+1](https://github.com/DagnyT/hardnet/blob/9588b957724c394df6b12c03ba2509df6118a7fe/code/dataloaders/TotalDataLoader.py#L19)

Seemingly,  assume that length of  datasets[0][1] and dataset[1][1]  is n1, and n2 respectively and (n1<n2). So, datasets[0][1][:n1]==datasets[1][1][:n1], but they are not the positive pairs. 
Is it right? the original line is a  bug ? @ducha-aiki  @DagnyT  @spongezhang

**Comment by DagnyT** [comment]
Hello, thank you, yes, it's a bug, technically it should have been like this:        
labels = torch.cat([labels, datasets[i][1]+torch.max(labels)+1] in such way to shift label indices.

**Comment by yunyundong** [comment]
How can I point to the newest version? what's the address? Thank you very much.

**Comment by DagnyT** [comment]
Thank you, yes, it was a bug in TotalDataLoader. Please, check the update.
